src/aat/resources/features/F-040/F-040_Later_Case_Read_By_Solicitor_1.td.json

```diff
@@ -0,0 +1,167 @@
+{
+  "title": "Get the case for read by a caseworker",
+  "_extends_": "F-040_Test_Data_Base",
+  "_guid_": "F-040_Later_Case_Read_By_Solicitor_1",
+  "specs": [
+    "to get the same case by Solicitor 1"
+  ],
+  "user": {
+    "username": "befta.caseworker.2.solicitor.1@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "pathVariables": {
+      "cid": "${[scenarioContext][parentContext][childContexts][Befta_Jurisdiction2_Default_Full_Case_Creation_Data][testData][actualResponse][body][id]}",
+      "uid": "[[DYNAMIC]]",
+      "jid": "BEFTA_JURISDICTION_2",
+      "ctid": "BEFTA_CASETYPE_2_1"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Length": "[[ANY]]",
+      "Content-Type": "[[ANY]]",
+      "Transfer-Encoding": "chunked"
+    },
+    "body": {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": 0,
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
+        "DocumentField4": {
+          "document_url": "http://dm-store:8080/documents/e56f0aea-4a5f-46d5-82d3-01b536420870",
```

Comment:
there is some url validation on the preview build which will reject this document_url. The domain needs to be changed from `dm-store:8080` to `dm-store-aat.service.core-compute-aat.internal` (so this url should be updated to `http://dm-store-aat.service.core-compute-aat.internal/documents/e56f0aea-4a5f-46d5-82d3-01b536420870`). Please can you update it here and all other occurrences in the PR